R/render.r

```diff
@@ -301,3 +303,64 @@ check_made_by <- function(first) {
   if (length(first) == 0L) return(FALSE)
   grepl("<!-- Generated by pkgdown", first, fixed = TRUE)
 }
+
+pkgdown_footer <- function(data, pkg) {
+  footer_components <- list(
+    authors = footer_authors(data),
+    pkgdown = footer_pkgdown(data)
+  )
+
+  # footer left
+  left_structure <- pkg$meta$footer$left$structure %||% c("authors")
+
+  left_components <- utils::modifyList(
```

Comment:
For the footer, I wonder if we should paste all the components together and then convert to markdown? That would give you a little bit more control.